packages/nextra-theme-docs/src/types.ts

```diff
@@ -1,13 +1,18 @@
+import React from 'react'
+
 export interface DocsThemeConfig {
+  projectLink?: string
+  github?: string
+  projectLinkIcon?: React.ReactNode | React.FC<{ locale?: string }>
   docsRepositoryBase?: string
   titleSuffix?:
-    | string
```

Comment:
why this change? according to types now specifying `titleSuffix` as string will be incorrect